From c190f641bd81d867ff98bdb4275d09ff4b7ddb4c Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 8 Apr 2024 18:45:58 +0200
Subject: [PATCH 0814/1695] dts: stm32: set tampers as a wakeup source for
 stm32mp257f-dk

Configure EXTI2 as TAMP wakeup parent.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git bff7e46da671ddb34d1441c2ec9293dc813c00bf dts: stm32: set tampers as a wakeup source for stm32mp257f-dk]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Ic4ccad60a54ba60eef304442d698db891c2bbe1d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/372212
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp257f-dk.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index d1c4b947a..732c03c5e 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -460,6 +460,8 @@
 
 &tamp {
 	status = "okay";
+	wakeup-source;
+	wakeup-parent = <&exti2>;
 	st,tamp-passive-precharge = <2>;
 	st,tamp-passive-nb-sample = <4>;
 	st,tamp-passive-sample-clk-div = <16384>;
-- 
2.47.2

